h2o-py/tests/testdir_algos/coxph/pyunit_pubdev_8945_coxph_heart_interactions_catNum_catCat_NumNum_NA_mojo.py

```diff
@@ -0,0 +1,59 @@
+import sys
+import tempfile
+
+sys.path.insert(1, "../../../")
+import h2o
+from tests import pyunit_utils
+from h2o.estimators.coxph import H2OCoxProportionalHazardsEstimator
+
+# this test makes sure the enum x enum, enum x num, num x num interactions work for coxph mojo
+def test_coxph_enum_num_interactions_orders(interaction_orders):
+    # bunch of random columns to be added to the dataset
+    data = h2o.import_file("https://s3.amazonaws.com/h2o-public-test-data/smalldata/coxph_test/heart_random_num_enum_NAs.csv")
```

Comment:
`pyunit_utils.locate("smalldata/coxph_test/heart_random_num_enum_NAs.csv")`

ideally we should use local smalldata

Reply:
Yes, they will run faster without having to goto AWS.  I changed it.